platform/mbed_stats.h

```diff
@@ -41,6 +47,19 @@ typedef struct {
     uint32_t alloc_fail_cnt;    /**< Number of failed allocations. */
 } mbed_stats_heap_t;
 
+/**
+ * struct mbed_stats_thread_t definition
+ */
+
+typedef struct {
+    uint32_t thread_id;             /**< Thread Object Identifier */
+    uint32_t thread_state;          /**< Thread Object State */
+    uint32_t thread_priority;       /**< Thread Priority */
+    uint32_t thread_stack_size;     /**< Thread Stack Size */
+    uint32_t thread_stack_space;    /**< Thread remaining stack size */
+    const char   *thread_name;      /**< Thread Object name */
+} mbed_stats_thread_t;
```

Comment:
Do we need to repeat thread_ prefix ?

`mbed_stats_thread_t` says  this contains tread stats. all members are related  to the thread (if not they would be a separate object?). Thus would be thread_stats.id/state/priority oppose to thread_stats.thread_id ?